readme.md

```diff
@@ -276,6 +276,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Dev Env](https://github.com/jondot/awesome-devenv)
 - [Dotfiles](https://github.com/webpro/awesome-dotfiles)
 - [Shell](https://github.com/alebcay/awesome-shell)
+- [Fish](https://github.com/fisherman/awesome-fish)
```

Comment:
Could you update the link to the new repository location: https://github.com/brj/awesome-fish and also indent the bullet point so it is forms under the Shell list.